### PR TITLE
Mudlet : new recipe

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by extrowerk** [comment]
It complains about some missing optional luarocks modules, but works ok.

File: game-mud/mudlet/mudlet-3.5.0.recipe [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	lib:libQt5Multimedia$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5OpenGL$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libyajl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libzip$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
```
Comment (korli) [inline game-mud/mudlet/mudlet-3.5.0.recipe]: Should come after REQUIRES.

File: game-mud/mudlet/mudlet-3.5.0.recipe [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	cmd:grep
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	cmd:sed
+	cmd:strip
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
```
Comment (Begasus) [inline game-mud/mudlet/mudlet-3.5.0.recipe]: REQUIRES_devel?

Comment (extrowerk) [reply game-mud/mudlet/mudlet-3.5.0.recipe]: what?

File: game-mud/mudlet/mudlet-3.5.0.recipe (hunk 1 above)
Comment (extrowerk) [reply game-mud/mudlet/mudlet-3.5.0.recipe]: thx.

**Pull request merged**